src/backend/pipeline/modules/mosaicing.py

```diff
@@ -70,3 +77,28 @@ def run(self, data: Data) -> Data:
             
         # Run the next module
         return super().run(data)
+    
+    def process(self, img: Mat) -> (np.ndarray, Mat):
+        """
+        Extract and process information from the orthomosaic image.
+
+        Args:
+            img: a Mat object containing the stitched image
+        
+        Returns:
+            tuple consisting of a np.ndarray and a new Mat 
+            representation
+        """
+        # collapse channels to ensure all are 0
+        collapsed = np.sum(img.get(), axis=2)
+        # create the mask
+        mask = np.where(collapsed == 0.0, 1, 0)
+        # expand the number of dimensions for concatenation
+        mask = np.expand_dims(mask, 2)
+        alpha_image = np.concatenate((img.get(), mask), axis=2)
+        return np.where(mask == 1, 0, 1), Mat(alpha_image, channels=[Channels.R, 
+                            Channels.G, Channels.B, Channels.A])
```

Comment:
You can fit the mask in a `cv2.Mat` 😃 - see the `RGBA` type, and save it as a `.png`.

Reply:
Ok, so I just put the mask in a *Mat* object for consistency (this is why we created the Mat object in the first place:))
I think this should be fine. Also added a GREYSCALE channel